changelog/58800.fixed

```diff
@@ -0,0 +1 @@
+Number of bugs in boto_sns to get it to pass newly added unit tests.
```

Comment:
**question** could this message be more specific?

Reply:
Yuppers.  Fixed it up to list everything, though, towncrier doesn't make formatting the list all that nice.